### DISCUSSION TITLE
Docker container or installation script broken?

### DISCUSSION THREAD

**Discussion by ddmee** [srid/emanote#257] [Q&A]
**Describe the bug**
Following the [docker installation instructions](https://emanote.srid.ca/start/install/docker) 

When I run that code, the docker container starts and immediately exists with exit code 137

```shell
docker ps -a
CONTAINER ID   IMAGE                             COMMAND                  CREATED          STATUS                       PORTS     NAMES
b23b2d893764   sridca/emanote                    "emanote -L /data ge…"   11 seconds ago   Exited (137) 7 seconds ago             nostalgic_maxwell
```

Further, docker logs return nothing. So it looks like the process has crashed very early.

```shell
docker logs nostalgic_maxwell 
```

**To Reproduce**

Code is 

```shell
cd <mypathtonotebook>
mkdir output.docker
docker run \
  -e LANG=C.UTF-8 -e LC_ALL=C.UTF-8 \
  --tmpfs /tmp \
  -v $PWD:/data \
  sridca/emanote emanote -L "/data" gen /data/output.docker
open ./output.docker/index.html
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
The file <mypathtonotebook>output.docker/index.html does not exist.
```

**Expected behavior**
I expect the docker container not to immediately exit.

**Screenshots**

**Desktop (please complete the following information):**
I'm on Mac OSX 12.0.1  with Docker version 20.10.10, build b485636

**Additional context**

Ideally I wanted to use `docker-compose`. But I can't even get the `docker run` example command to run. So I suspect something has recently changed that means the run command is incorrect or something.

**Comment by srid** [comment]
Was this working before? I don't use docker on M1 macOS, so I can't tell if it even worked on it at some point.

**Comment by ddmee** [comment]
This is the first time I'm trying emanote so I have no idea if it was working before.

**Reply by srid** [reply]
I do know that the docker container works on Linux, as the entirety of https://emanote.srid.ca/ is generated using it in GitHub Actions. The issue appears to be macOS-specific; would be nice if somebody can diagnose this. The Nix code for building the image is here: https://github.com/srid/emanote/blob/143b53cc4d5fdcb7580f6db2da3725575d7dcc15/docker.nix#L26-L45
